tasks/broad/Qc.wdl

```diff
@@ -218,23 +218,23 @@ task ConvertSequencingArtifactToOxoG {
   Int java_memory_size = (memory_size - 1) * 1000
 
   command {
-      input_base=$(dirname ~{pre_adapter_detail_metrics})/~{base_name}
-      java -Xms~{java_memory_size}m -Dpicard.useLegacyParser=false \
-        -jar /usr/picard/picard.jar \
-        ConvertSequencingArtifactToOxoG \
-        --INPUT_BASE $input_base \
-        --OUTPUT_BASE ~{base_name} \
-        --REFERENCE_SEQUENCE ~{ref_fasta}
-    }
-    runtime {
-      docker: "us.gcr.io/broad-gotc-prod/picard-cloud:2.22.3"
-      memory: "~{memory_size} GiB"
-      disks: "local-disk " + disk_size + " HDD"
-      preemptible: preemptible_tries
-    }
-    output {
-      File oxog_metrics = "~{base_name}.oxog_metrics"
-    }
+    input_base=$(dirname ~{pre_adapter_detail_metrics})/~{base_name}
+    java -Xms~{java_memory_size}m -Dpicard.useLegacyParser=false \
```

Comment:
do we need this `-Dpicard.useLegacyParser=false` ?

Reply:
Not anymore.  That was my doing in trying to start using the barclay syntax.  Now that we have the UberParser in place, I'll remove those.